tools-ui/src/main/java/com/hedera/hashgraph/client/ui/utilities/TimeFieldSet.java

```diff
@@ -0,0 +1,396 @@
+/*
+ * Hedera Transaction Tool
+ *
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.hashgraph.client.ui.utilities;
+
+import com.hedera.hashgraph.client.core.json.Timestamp;
+import javafx.scene.control.DateCell;
+import javafx.scene.control.DatePicker;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import javafx.scene.input.KeyCode;
+import javafx.scene.layout.HBox;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.TimeZone;
+
+public class TimeFieldSet {
+	private static final Logger logger = LogManager.getLogger(TimeFieldSet.class);
+
+	DatePicker datePicker;
+	TextField hours;
+	TextField minutes;
+	TextField seconds;
+	TextField nanos;
+
+	HBox timeZoneBox;
+	TimeZone timeZone;
+
+	Label utcLabel;
+	Label errorLabel;
```

Comment:
I probably did not review the PR for this change. But these fields should come with `private` right?

Reply:
Gosh, yes! I don't know how that escaped me. I will make the change now. Great catch, thanks